### ISSUE TITLE
[jsconfig] compilerOptions.target should be case insensitive

### ISSUE THREAD

**Issue by aeschli** [Microsoft/TypeScript#9826]
http://json.schemastore.org/jsconfig defines compilerOptions.target as
```
     "target": {
              "description": "Specifies which default library (lib.d.ts) to use. When down-level compiling, specifies the code being generated.",
              "enum": [ "es3", "es5", "es6", "es2015" ],
              "default": "es6"
            },
```
Problem is the values are actually in-case sensitive, so 'ES3', 'ES5'.. are also valid. Existing projects that use the capitalized variant now get a warning.
It would be great if this could be done the same ways as in tsconfig.json:
```
     "target": {
              "description": "Specify ECMAScript target version. Permitted values are 'es3', 'es5', or 'es2015'.",
              "type": "string",
              "pattern": "^([eE][sS]([356]|(2015)))$",
              "default": "es3"
            }
```

**Comment by mhegazy** [comment]
PRs to http://json.schemastore.org are welcomed.

**Comment by aeschli** [comment]
@mhegazy No problem: https://github.com/SchemaStore/schemastore/pull/203

**Comment by mhegazy** [comment]
Fixed in SchemaStore/schemastore#203. thanks @aeschli
